RUFAS/data_collection_app_updater.py

```diff
@@ -101,11 +84,15 @@ def _rewrite_schemas(self) -> list[Path]:
         properties: dict[str, Any] = self._im.meta_data["properties"]
 
         schema_paths = []
+
         for key in properties.keys():
-            if key not in PROPERTIES_TO_CREATE_SCHEMA_FOR:
+            if not properties[key]["data_collection_app_compatible"]:
```

Comment:
```suggestion
            data_collection_app_compatible = properties[key].get("data_collection_app_compatible", False)
            if not data_collection_app_compatible:
```
So that if for som reason, the "data_collection_app_compatible" key is not specified in the metadata properties, it wouldn't crash the simulation.